intelmq/bots/BOTS

```diff
@@ -4,6 +4,8 @@
             "description": "Fileinput collector fetches data from a file.",
             "module": "intelmq.bots.collectors.file.collector_file",
             "parameters": {
+                "chunk_replicate_header": true,
```

Comment:
Correctly if I'm wrong but I think we need to replicate this solution to all collectors, right?